src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -1315,6 +1315,7 @@ private static bool IsMemberDeclaration(SyntaxNode node)
                 case SyntaxKind.PropertyDeclaration:
                 case SyntaxKind.GetAccessorDeclaration:
                 case SyntaxKind.SetAccessorDeclaration:
+                case SyntaxKind.InitAccessorDeclaration:
```

Comment:
Reverting, and will file an issue to track test coverage for this method and adding the init accessor if necessary.

```suggestion
```